lib/request-base.js

```diff
@@ -3,6 +3,19 @@
  */
 
 /**
+ * Root reference for environment-specific globals.
+ */
+
+var root;
+if (typeof window !== 'undefined') { // Browser window
+  root = window;
+} else if (typeof self !== 'undefined') { // Web Worker
+  root = self;
+} else { // Other environments (node or other)
+  root = {FormData: require('form-data')};
```

Comment:
Good point. I checked and it looks like form-data is properly browserified to use the built-in `FormData` on the browser:

https://github.com/form-data/form-data/blob/master/lib/browser.js (their package.json file has the proper "browser" property as well).

I'm OK with this approach overall in dealing with node/browser discrepancies. We'll have a few others to address soon enough like btoa, Buffer, etc.